src/cript/api/utils/save_helper.py

```diff
@@ -75,15 +91,13 @@ def _fix_node_save(api, node, response, save_values: _InternalSaveValues) -> _In
                 try:
                     save_values.suppress_attributes[str(duplicate_node.uuid)] = set(search_dict.keys())
                 except KeyError:
-                    save_values.suppress_attributes[str(duplicate_node.uuid)].add(set(search_dict.keys()))
+                    save_values.suppress_attributes[str(duplicate_node.uuid)].add(set(search_dict.keys()))  # type: ignore
```

Comment:
This needs comments as to what is going on

Reply:
I added some comments.